src/averageAges.js

```diff
@@ -37,7 +44,22 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const filteredWomen = people
+    .filter(person => {
+      const { sex, name } = person;
+
+      if (withChildren === undefined) {
+        return sex === 'f';
+      }
```

Comment:
This check is redundant as the one below can handle it when `withChildren` is undefined

Reply:
@dinaminahi I missed that moment, thanks